brille_spinwaves/run_matlab_test.ps1

```diff
@@ -0,0 +1,37 @@
+. $PSScriptRoot/../powershell_scripts/powershell_helpers.ps1 <# Imports:
+  Get-From-Registry
+#>
+
+<#
+  .SYNOPSIS
+    Runs a Matlab script to run integration tests
+  .DESCRIPTION
+    Gets the Matlab command from registry and runs it
+  .NOTES
+    Required environment variables:
+      MATLAB_VERSION - Matlab version to use e.g. 2019b
+#>
+
+$ErrorActionPreference = 'Stop'
+
+$MATLAB_VERSION_MAP = @{
+  '2018a' = '9.4';
+  '2018b' = '9.5';
+  '2019a' = '9.6';
+  '2019b' = '9.7';
+  '2020a' = '9.8';
+  '2020b' = '9.9';
```

Comment:
A note saying why later versions are not listed here (?not yet on Jenkins?) or earlier versions (?no longer fully supported or available?) would be helpful. What does one have to do to activate 2021b?

Reply:
Ah... this file should be deleted. We're using the file in the [shared library repo](https://github.com/pace-neutrons/PACE-jenkins-shared-library/blob/main/powershell_scripts/execute_matlab_command.ps1) - but that file itself should be edited to include R2021a.